abc.templates/infra/spec.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: 'cli.abcxyz.dev/v1alpha1'
+kind: 'Template'
+
+desc: 'Generate the infrastructure necessary to run GitHub Token Minter'
+
+inputs:
+  - name: 'project_id'
```

Comment:
I think all of these APIs technically accept a Project ID _or_ Project Number (except WIF, which only supports number). I wonder if we should support both?

Reply:
@sethvargo

Reply:
Sticking with just project ID is fine.